### ISSUE TITLE
False positive on timeouts attribute in aws_route_table

### ISSUE THREAD

**Issue by wbeuil** [snyk/driftctl#1500]
**Description**
One user found a useless `timeouts` block in its scan for `aws_route_table` resource. We should ignore it to avoid false positive drifts.

**How to reproduce**
Create a `aws_route_table` with a timeouts block.

**Possible Solution**
```go
func initAwsRouteTableMetaData(resourceSchemaRepository resource.SchemaRepositoryInterface) {
	resourceSchemaRepository.SetNormalizeFunc(AwsAmiResourceType, func(res *resource.Resource) {
		val := res.Attrs
		val.SafeDelete([]string{"timeouts"})
	})
}
```

**Comment by karniwl** [comment]
for some reason the issue isn't tagged with the PR https://github.com/snyk/driftctl/pull/1545? @chdorner-snyk is this resolved?

**Comment by craigfurman** [comment]
Please reopen if this isn't indeed resolved by #1545
